.storybook/custom/components/IfBlock.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export const IfBlock = ({condition, children}) => {
+    if (condition) {
+        return <>{children}</>
+    }
+    return <></>;
```

Comment:
we can return `null` if we don't want to render anything.

Reply:
ha, thanks for catch, I did not knew that